Helper/Setup.php

```diff
@@ -140,6 +130,7 @@ public function upgradeData(string $mode, ModuleDataSetupInterface $setup, Modul
             }
         } catch (Throwable $e) {
             $this->loggingClient->error("Protect $mode failed", $e);
+            throw $e;
```

Comment:
rethrowing the error here so that Magento knows the install didn't succeed.  The actual error was that `accessToken` doesn't exist in `$installResult`; maybe there's a better error message we could pass (e.g., an error that comes back in `$installResult`)?